compiler/src/iree/compiler/Codegen/LLVMCPU/ConvertToLLVM.cpp

```diff
@@ -959,7 +959,15 @@ void ConvertToLLVMPass::runOnOperation() {
   //   multiply or add.
   //
   // TODO(bjacob): Use a lowering that uses specific ARM/X86 intrinsics.
-  tosa::populateTosaRescaleToArithConversionPatterns(&patterns);
+  bool use32BitImpl = false;
```

Comment:
Based on the comment right above this code, I decided to change the policy and always use the 64-bit lowering by default. That will preserve the existing behavior.